src/views/requestAccess/requestAccess.js

```diff
@@ -104,23 +155,54 @@ export const RequestAccess = (props) => {
   }, []);
 
   return (
-    <>
-      <div className='main-container'>
-        <div className="page-title"> Request for Access </div>
-        <div className="sub-title"> CONTACT </div>
-        <InfoField label={"First Name"} changeHandler={setFirstName} info={firstName} />
-        <InfoField label={"Last Name"} changeHandler={setLastName} info={lastName} />
-        {/* <InfoField label={"Phone"} info={data['phone']} /> */}
-        <InfoField label={"Email"} changeHandler={setEmail} info={email} />
-        <hr className="line" ></hr>
-        <div className="sub-title sub-title-padding"> ASSIGN ROLE </div>
-        <RoleDropDown selectionOptions={selectionOptions} selectionHandler={selectionHandler} />
-        <div className="mt-2">
-          <button className="button is-small is-rounded is-primary mx-4" onClick={() => grantAccess(currentSelection, fName, lName, emailAddress, id)} disabled={currentSelection === ""}> GRANT ACCESS </button>
+    <div className='main-container'>
+      <div className="page-title"> Request for Access </div>
+      <div className="sub-title"> CONTACT </div>
+      <InfoField label={"First Name"} changeHandler={setFirstName} info={firstName} />
+      <InfoField label={"Last Name"} changeHandler={setLastName} info={lastName} />
+      {/* <InfoField label={"Phone"} info={data['phone']} /> */}
+      <InfoField label={"Email"} changeHandler={setEmail} info={email} />
+      <hr className="line" ></hr>
+      <div className="sub-title sub-title-padding"> ASSIGN ROLE </div>
+      <RoleDropDown selectionOptions={selectionOptions} selectionHandler={selectionHandler} />
+      <h1 className="section-title">PROPERTY</h1>
```

Comment:
The schema says otherwise, but I feel "properties" is more appropriate:

`<h1 className="section-title">PROPERTIES</h1>`